packages/blockchain-wallet-v4-frontend/src/data/components/nfts/slice.ts

```diff
@@ -59,7 +60,22 @@ const initialState: NftsStateType = {
     matchingOrder: Remote.NotAsked,
     offerToCancel: null,
     step: NftOrderStepEnum.SHOW_ASSET
-  }
+  },
+  sellOrder: Remote.NotAsked,
+  status: {
```

Comment:
actually since this is remote data (requested from an API) the initial state should be Remote.NotAsked